website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -106,3 +109,53 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt --help`: Lists the commands available for dbt<br />
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
+
+
+## FAQs
+<expandable alt_header="How to create a .dbt directory and move your file">
+
+If you've never had a `.dbt` directory, you should perform the following recommended steps to create one. If you already have `.dbt` directory, then you should be able to move the `dbt_cloud.yml` file into it.
+
+<Tabs>
+<TabItem value="Create a .dbt directory">
+
+  1. Clone your dbt project repository locally.
+  2. Use the `cd` command to change directories and navigate to the root of your filesystem. (For example, if you're a Mac user it might be `Macintosh HD -> Users -> YOUR_USERNAME`):
+
+     ```bash
+     cd ~/Users
+      ```
+  3. Use the `mkdir` command followed by the name of the folder you want to create. To create a `.dbt` folder, use:
+
+     ```bash
+     mkdir .dbt
+     ```
+
+This will create a `.dbt` folder in the current directory. 
+
+For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. However, to view hidden files and folders press Command + Shift + G. 
+
+</TabItem>
+
+<TabItem value="Move the dbt_cloud.yml file">
+
+**On Mac or Linux**
+- In your command line, use the `mv` command to move your `dbt_cloud.yml` file into the `.dbt` directory. If you've just downloaded the `dbt_cloud.yml` file and it's in your Downloads folder, the command might look something like this:
+
+```bash
+mv ~/Downloads/dbt_cloud.yml ~/.dbt/dbt_cloud.yml
+```
+
+**On Windows**
+- In your command line, use the move command. Assuming your file is in the Downloads folder, the command might look like this:
+
+```bash
+move %USERPROFILE%\Downloads\dbt_cloud.yml %USERPROFILE%\.dbt\dbt_cloud.yml
+```
+
+</TabItem>
+</Tabs>
+
+This command moves the `dbt_cloud.yml` from the Downloads folder to the `.dbt` folder. If your `dbt_cloud.yml` file is located elsewhere, adjust the path accordingly.
```

Comment:
Why does the `.dbt` folder name get code formatted but not the Downloads folder name? Do we use code formatting to call attention to names?

Reply:
yea it was trying to make it clear since the .dbt folder is a hidden folder. could also format 'downloads' too